image_provisioner/playbooks/roles/docker-spread-images/tasks/main.yaml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Creates directory
+  file:
+    path: "{{ image_dir }}"
+    state: directory
+
+- name: Rsync docker images
+  synchronize:
+    src: "{{ image_dir }}"
+    dest: "{{ image_dir }}"
+  delegate_to: "{{ jump_node_ip }}"
+    
+- name: Register all images
+  find: 
+    path: "{{ image_dir }}"
+    file_type: file
+    recurse: yes
+  register: image_files
+
+- debug: var=image_files
+
+- name: Docker load images
+  command: docker load -i {{ item.path }}
+  with_items: "{{ image_files.files }}"
```

Comment:
The this load task only puts stress on the local storage, do you think that is an issue? I haven't seen any errors in any of the deploys so far?